api/src/main/java/org/pmiops/workbench/db/model/User.java

```diff
@@ -29,6 +30,23 @@
 @Entity
 @Table(name = "user")
 public class User {
+
+  /**
+   * This is a Gson compatible class for encoding a JSON blob which is stored in MySQL. This
+   * represents cluster configuration overrides we support on a per-user basis for their notebook
+   * cluster. Corresponds to Leonardo's MachineConfig model. All fields are optional.
+   *
+   * Any changes to this class should produce backwards-compatible JSON.
+   */
+  public static class ClusterConfigOverride {
```

Comment:
Agreed - renamed. Sorry, wasn't ignoring this - wanted to make sure we reached closure on the other comment thread first since it had implications here.